packages/vision/src/index.js

```diff
@@ -73,7 +73,7 @@ function visionV1(opts) {
   // Create the image annotator client with the provided options.
   var client = gapic.v1(opts).imageAnnotatorClient(opts);
   if (is.undefined(client.annotateImage)) {
-    Object.assign(client.constructor.prototype, helpers('v1'));
```

Comment:
Same prototype prob here. In this case it was actually breaking the grpc validation. Not sure how this used to work